settings.py

```diff
@@ -174,7 +174,7 @@
 # Use this when OPENWRAP_SETUP_TYPE = "ADPOD"
 # This parameter will be used to set creative level targeting. ex s1_pwtdur = 10
 # Represents the video length parameter on UI.
-VIDEO_LENGTHS = [10,15]
+VIDEO_LENGTHS = [10]
```

Comment:
Why is this changed?

Reply:
This is just a sample config. I have set it to empty now , what kind of values are expected is mentioned in the comment above